static/app/views/dashboardsV2/controls.tsx

```diff
@@ -175,7 +175,7 @@ class Controls extends React.Component<Props> {
   }
 }
 
-const DashboardEditFeature = ({
+export const DashboardEditFeature = ({
```

Comment:
```suggestion
const DashboardEditFeature = ({
```

we can remove the export, as this component is only used in the controls component